src/main/java/edu/harvard/iq/dataverse/RoleAssignment.java

```diff
@@ -71,13 +71,21 @@
     @Column(nullable = true)
     private String privateUrlToken;
 	
+    @Column(nullable = true)
+    private Boolean privateUrlAnonymizedAccess;
+	
```

Comment:
We need a sql update script for this new field:

ALTER TABLE RoleAssignment ADD COLUMN IF NOT EXISTS privateUrlAnonymizedAccess BOOLEAN;

UPDATE RoleAssignment SET privateUrlAnonymizedAccess = false WHERE privateUrlAnonymizedAccess IS NULL;

Reply:
Good catch. I don't think the second line is needed (at least it works for QDR without it).